### PR TITLE
enh(service-mapping): adapt screenshots to 20.10, wording and linting

### PR DESCRIPTION
**Pull request by cgagnaire**
## Description

Adapt screenshots to 20.10.
Fix some wording and linting, mostly in french.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x (master)

### CODE REVIEW

File: fr/service-mapping/ba-management.md [hunk 1]
```diff
@@ -434,12 +468,11 @@ Supprimer une vue métier:
...
-  --------------------------|-------------------------------------------------------------
-  Nom                       | Nom de la vue métier
-  Description               | Description de la vue métier
-  Activité(s) métier        | La ou les activité(s) métier associées à la BV concernée(s)       
-  Groupes d'accès autorisés | Le ou les groupes d'accès pouvant consulter la vue métier et les activités métier contenues
+| Colonne                   | Description                                                                                 |
+|---------------------------|---------------------------------------------------------------------------------------------|
+| Nom                       | Nom de la vue métier                                                                        |
+| Description               | Description de la vue métier                                                                |
+| Activité(s) métier        | La ou les activité(s) métier associées à la BV concernée(s)                                 |
```
Comment (kduret) [inline fr/service-mapping/ba-management.md:477]:
```suggestion
| Activité(s) métier        | La ou les activité(s) métier associée(s) à la BV concernée                                  |
```

File: fr/service-mapping/ba-monitoring.md [hunk 2]
```diff
@@ -18,17 +18,18 @@ visualisées à la page *Monitoring > Business Activity > Monitoring*
 La page principale de monitoring est une console présentant les
 informations essentielles concernant le status des BA en temps réel.
 
-Un utilisateur non administrateur verra uniquement les BA appartenant aux BV lié à son groupe d'accès.
+Un utilisateur non administrateur verra uniquement les BA appartenant
+aux BV lié à son groupe d'accès.
```
Comment (kduret) [inline fr/service-mapping/ba-monitoring.md:22]:
```suggestion
aux BV liées à son groupe d'accès.
```

File: fr/service-mapping/ba-monitoring.md [hunk 3]
```diff
@@ -47,15 +48,20 @@ La vue détaillée d'une BA est divisée en 7 parties.
...
+    déployer les niveaux inférieurs, zoomer et se déplacer dans le
+    graphique.
 4.  Onglet contenant les graphiques de niveau de services de la BA
 5.  Onglet contenant un tableau listant tous les KPIs de la BA
 6.  Menu déroulant permettant de modifier la BA
 7.  Lien vers la page de reporting de la BA
 
-Le tableau des KPI affichent en temps réel les informations sur leurs état mais aussi les impacts sur la BA.
+Le tableau des KPI affichent en temps réel les informations sur leurs
+état mais aussi les impacts sur la BA.
```
Comment (kduret) [inline fr/service-mapping/ba-monitoring.md:64]:
```suggestion
états mais aussi les impacts sur la BA.
```

File: fr/service-mapping/ba-management.md [hunk 4]
```diff
@@ -285,111 +313,117 @@ indicateurs à la fois :
...
- |  **Colonne**     | **Description**                                     
- |------------------|---------------------------------------------------------
- | .ssv File        | Ouvrir l'explorateur de fichiers
- | KPI Type         | Type de KPI à charger
- | Format           |  Aide sur le format du fichier SSV à utiliser en fonction du type de KPI choisi
- | Manual filling   |  Possibilité de compléter ce champ plutôt que de charger un fichier
+| Colonne            | Description                                                                    |
+|--------------------|--------------------------------------------------------------------------------|
+| Fichier .ssv       | Ouvrir l'explorateur de fichiers                                               |
+| Tyoe de KPI        | Type de KPI à charger                                                          |
```
Comment (cgagnaire) [inline fr/service-mapping/ba-management.md:392]:
```suggestion
| Type de KPI        | Type de KPI à charger                                                          |
```

**Review by kduret — APPROVED** [review]

**Review by Thebarda — CHANGES_REQUESTED** [review]

File: fr/service-mapping/ba-settings.md [hunk 5]
```diff
@@ -36,61 +37,41 @@ pré-remplir les champs lors de la bascule de la configuration en mode
...
 
-Les « User Settings » sont des options personnalisées qui sont propres à
-chaque profil d'utilisateur. Ces paramètres concernent surtout les
-effets visuels du module, à savoir les graphiques.
+## Paramètres utilisateur
 
-    Monitoring > Business Activity > User Settings
+Les **Paramètres utilisateur** sont des options personnalisées qui sont
+propres à chaque profil d'utilisateur, et peuvent être configurées dans
+le menu `Configuration > Activités métiers > Paramètres utilisateurq`.
```
Comment (cgagnaire) [inline fr/service-mapping/ba-settings.md:68]:
```suggestion
le menu `Configuration > Activités métiers > Paramètres utilisateurs`.
```

**Review by Thebarda — APPROVED** [review]

File: fr/service-mapping/ba-settings.md [hunk 6]
```diff
@@ -9,25 +9,26 @@ Le module **Centreon BAM** offre une série d'option permettant de
 déployer vos BA plus facilement et d'avoir une visualisation
 personnalisée de vos graphiques.
 
-Les « Default settings » ont pour but de pré-remplir certains des champs
+Les **Paramètres par défaut** ont pour but de pré-remplir certains des champs
 de saisie lors de l'ajout de nouvelles BA et de nouveaux indicateurs.
 
-    Supervision > Activité Métier > Paramètres par défaut 
+Ils sont configurable depuis le menu
+`Supervision > Activité Métier > Paramètres par défaut`.
```
[the remark this answers is not in the document]
Comment (cgagnaire) [reply fr/service-mapping/ba-settings.md]: Actually we try to use french menu and buttons in french docs.
Same for screenshot, which is not always respected as it is really painful to always create 2 sets of screenshots, even more when its gif or videos!
So I fixed this doc to use only french menu.

**Pull request merged**